### PR TITLE
Add solid-crud tests

### PR DESCRIPTION
**Pull request by michielbdejong**
Currently failing 73/73 tests.

### CODE REVIEW

File: run-solid-test-suite.sh [hunk 1]
```diff
@@ -24,7 +25,8 @@ docker logs server
 echo Getting cookie...
 export COOKIE="`docker run --cap-add=SYS_ADMIN --network=testnet --env-file /tmp/env-vars-for-test-image.list cookie`"
 echo "Running webid-provider tests with cookie $COOKIE"
-docker run --rm --network=testnet --env COOKIE="$COOKIE" --env-file /tmp/env-vars-for-test-image.list webid-provider
+# docker run --rm --network=testnet --env COOKIE="$COOKIE" --env-file /tmp/env-vars-for-test-image.list webid-provider
```
Comment (Potherca) [inline run-solid-test-suite.sh]: Shouldn't this be uncommented? (i.e. shouldn't these be made to pass as well?)

Comment (michielbdejong) [reply run-solid-test-suite.sh]: sure, it's just commented out temporarily to save you time when you run this script.

**Review by Potherca — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -21,3 +21,4 @@ RUN php /install/composer.phar install --no-dev --prefer-dist
 COPY site.conf /etc/apache2/sites-enabled/site.conf
 RUN chown www-data /app/config
 EXPOSE 443
+RUN chown www-data:www-data config
```
Comment (michielbdejong) [inline Dockerfile]: Isn't this a duplication of line 22?

**Comment by michielbdejong** [comment]
The webid-provider tests are currently failing in this branch:
```shTests:
       9 failed, 26 passed, 35 total

**Comment by michielbdejong** [comment]
The webid-provider tests passing in the ./run-solid-test-suite.sh script (again) now.
So that problem must be with how the solid test suites executes.

**Comment by michielbdejong** [comment]
Should be passing 61/61 now

**Pull request merged**